images/makeproject/project/templates/onechunk_in

```diff
@@ -0,0 +1,13 @@
+<input_template>
+    <file_info>
+    </file_info>
+    <workunit>
+        <file_ref>
+            <open_name>jf_MD5</open_name>
+            <copy_file/>
+        </file_ref>
+        <rsc_fpops_est>2.8e13</rsc_fpops_est>
+        <rsc_fpops_bound>2.8e20</rsc_fpops_bound>
+        <delay_bound>259200</delay_bound>
```

Comment:
Same thing with the delay_bound. Though it's redundant to put it both in the command and in the template.
```suggestion
        <delay_bound>1209600</delay_bound>
```